src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.trim().split(';');
```

Comment:
Consider adding a check to ensure `sourceString` is not null or undefined before calling `trim()` and `split()`. This will prevent potential runtime errors if the function is called with invalid input.